plugins/auth-backend/src/providers/types.ts

```diff
@@ -162,84 +165,13 @@ export type AuthResponse<ProviderInfo> = {
 };
 
 /**
- * User identity information within Backstage.
- *
- * @public
- */
-export type BackstageUserIdentity = {
-  /**
-   * The type of identity that this structure represents. In the frontend app
-   * this will currently always be 'user'.
-   */
-  type: 'user';
-
-  /**
-   * The entityRef of the user in the catalog.
-   * For example User:default/sandra
-   */
-  userEntityRef: string;
-
-  /**
-   * The user and group entities that the user claims ownership through
-   */
-  ownershipEntityRefs: string[];
-};
-
-/**
- * A representation of a successful Backstage sign-in.
- *
- * Compared to the {@link BackstageIdentityResponse} this type omits
- * the decoded identity information embedded in the token.
- *
- * @public
- */
-export interface BackstageSignInResult {
-  /**
-   * An opaque ID that uniquely identifies the user within Backstage.
-   *
-   * This is typically the same as the user entity `metadata.name`.
-   *
-   * @deprecated Use the `identity` field instead
-   */
-  id: string;
-
-  /**
-   * The entity that the user is represented by within Backstage.
-   *
-   * This entity may or may not exist within the Catalog, and it can be used
-   * to read and store additional metadata about the user.
-   *
-   * @deprecated Use the `identity` field instead.
-   */
-  entity?: Entity;
-
-  /**
-   * The token used to authenticate the user within Backstage.
-   */
-  token: string;
-}
-
-/**
- * The old exported symbol for {@link BackstageSignInResult}.
+ * The old exported symbol for {@link @backstage/plugin-auth-node#BackstageSignInResult}.
  *
  * @public
- * @deprecated Use the {@link BackstageSignInResult} instead.
+ * @deprecated Use the {@link @backstage/plugin-auth-node#BackstageSignInResult} instead.
  */
 export type BackstageIdentity = BackstageSignInResult;
```

Comment:
Could just remove this one no? Since the others are gone anyway. Been deprecated for quite a while now

Reply:
aight, removed and noted in the changeset